### PR TITLE
chore(other): remove obsolete version element from all docker yaml files

### PR DESCRIPTION
**Pull request by mahula**
## 🍰 Pullrequest
Thank you for the hint @roschaefer - this was on the todo list for a while.
Since the version element in docker yaml files [is obsolete now](https://github.com/compose-spec/compose-spec/blob/master/04-version-and-name.md), it can be removes ti avoid distracting warnings.

### Issues

- blocked by #7604

### CODE REVIEW

File: docker-compose.maintenance.yml [hunk 1]
```diff
@@ -1,8 +1,6 @@
 # Todo: !!! This file seems related to our old maintenance worker for MongoDB and has to be refactored in case of using it !!!
 
-version: "3.4"
-
-services:
+ervices:
```
Comment (roschaefer) [inline docker-compose.maintenance.yml:3]:
```suggestion
services:
```

:sob: this will be a "Request changes"

**Review by roschaefer — CHANGES_REQUESTED** [review]
typo

So close, but I have to request changes because of the typo.

**Review by Tirokk — DISMISSED** [review]
Cool @roschaefer !
Looks good.

Therefore I approve 🚀💫

**Comment by Tirokk** [comment]
Sorry, seems that coordinates of MapBox have changed again a little bit.
Just adjust them in the tests or do a separate PR to fix it. @mahula

**Comment by mahula** [comment]
> Sorry, seems that coordinates of MapBox have changed again a little bit. Just adjust them in the tests or do a separate PR to fix it. @mahula
> 
> https://github.com/Ocelot-Social-Community/Ocelot-Social/actions/runs/9662278558/job/26651955956?pr=7564

Please open a separate bug issue @Tirokk , since this is not related to this pull request.
And with time at hand someone ofus will take care of it.

**Review by Tirokk — APPROVED** [review]
Sorry, haven't remembered that you have already done @mahula

**Pull request merged**